specs/processes.table

```diff
@@ -44,6 +44,7 @@ extended_schema(DARWIN, [
     Column("uppid", BIGINT, "The 64bit parent pid that is never reused. Returns -1 if we couldn't gather them from the system."),
     Column("cpu_type", INTEGER, "Indicates the specific processor designed for installation."),
     Column("cpu_subtype", INTEGER, "Indicates the specific processor on which an entry may be used."),
+    Column("kind", TEXT, "Indicates the architecture of the process, apple or intel."),
```

Comment:
Hrm. I like the idea, but this feels like it's mostly duplicating `cpu_type`, but `cpu_type` is annoyingly useless, since it's a system constant. 

Which makes me wonder about whether to change `cpu_type` to a string.

Or name it `arch`